src/components/SingleChoiceListItem.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react'
+import { View } from 'react-native'
+import { capitalizeFirstLetter, getArticleColor } from '../utils/helpers'
+import { ARTICLES } from '../constants/data'
+import { COLORS } from '../constants/colors'
+import styled from 'styled-components/native'
+
+const StyledContainer = styled.View`
+  height: 55px;
+  margin-bottom: 5px;
+  border-radius: 2px;
+  border-width: 1px;
+  border-color: ${COLORS.lunesBlackUltralight};
+  border-style: solid;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-items: center;
+`
+
+const StyledContainerClicked = styled(StyledContainer)`
+  background-color: ${COLORS.lunesBlack};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
+
+const StyledContainerCorrect = styled(StyledContainer)`
+  background-color: ${COLORS.lunesFunctionalCorrectDark};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
+
+const StyledContainerIncorrect = styled(StyledContainer)`
+  background-color: ${COLORS.lunesFunctionalIncorrectDark};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
+
+const StyledArticle = styled.Text`
+  width: 37px;
+  height: 21px;
+  font-size: 14px;
+  font-weight: normal;
+  border-radius: 10px;
+  color: ${COLORS.lunesGreyDark};
+  font-family: SourceSansPro-Regular;
+  overflow: hidden;
+  text-align: center;
+  margin-right: 7px;
+  margin-left: 11px;
+  align-self: flex-start;
+  margin-top: 17px;
+  background-color: ${(props: any) => getArticleColor(props.article)};
+`
+
+const StyledArticleClicked = styled(StyledArticle)`
+  color: ${COLORS.lunesBlack};
+  background-color: ${COLORS.lunesWhite};
+`
+
+const StyledArticleCorrect = styled(StyledArticle)`
+  color: ${COLORS.lunesFunctionalCorrectDark};
+  background-color: ${COLORS.lunesBlack};
+`
+
+const StyledArticleIncorrect = styled(StyledArticle)`
+  color: ${COLORS.lunesFunctionalIncorrectDark};
+  background-color: ${COLORS.lunesBlack};
+`
+
+const StyledWord = styled.Text`
+  height: 18px;
+  font-family: SourceSansPro;
+  font-size: 14px;
+  font-weight: normal;
+  font-style: normal;
+  color: ${COLORS.lunesGreyDark};
+`
+const StyledWordClicked = styled(StyledWord)`
+  color: ${COLORS.lunesWhite};
+`
+
+const StyledWordEvaluated = styled(StyledWord)`
+  color: ${COLORS.lunesBlack};
+`
+
+export interface ISingleChoiceListItemProps {
+  word: string
+  article: string
+  correct: boolean
+  selected: boolean
+  addOpacity: boolean
+}
+
+const SingleChoiceListItem = ({ word, article, correct, selected, addOpacity }: ISingleChoiceListItemProps) => {
+  const [Container, setContainer] = useState<styled.Component>(StyledContainer)
+  const [Article, setArticle] = useState<styled.Component>(StyledArticle)
+  const [Word, setWord] = useState<styled.Component>(StyledWord)
+
+  useEffect(() => {
```

Comment:
I think this timeout is not needed at all though. Normally, the other color (blue) should only be shown while tapping on the item and afterwards either green or red should be displayed according to the fact whether it is correct or not.

Reply:
But wouldn`t the color blue be instantly overwritten if there was no timeout? Or do you mean to display the blue color as long as the finger still touches the button?

Reply:
The latter, yes. I think that is the default behavior.

Reply:
I have the same opinion as @klinzo.